### PR TITLE
Rewrite multiple OSError aliases in the same `except` block

### PR DESCRIPTION
**Pull request by mxr**
Brought up in #141

### CODE REVIEW

File: pyupgrade.py [hunk 1]
```diff
@@ -1880,6 +1889,70 @@ def _replace(i, mapping, node):
...
+                        right == 'error'
+                ):
+                    if not seen_os_error:
+                        args.append('OSError')
+                    seen_os_error = True
+                elif (
+                        left in visitor.OS_ERROR_ALIASES and
+                        part == right == ''
+                ):
+                    if not seen_os_error:
```
Comment (asottile) [inline pyupgrade.py]: could this be simplified with a `set` and `list`?

File: tests/oserror_aliases_test.py [hunk 2]
```diff
@@ -8,6 +8,72 @@
...
+            '    pass\n',
+
+            'try:\n'
+            '    pass\n'
+            'except OSError:\n'
+            '    pass\n',
+        ),
+    ),
+)
+def test_fix_oserror_aliases_try(alias, tpl, expected):
```
Comment (asottile) [inline tests/oserror_aliases_test.py]: can you add a test where the tuple spans multiple lines?

File: pyupgrade.py (hunk 1 above)
Comment (mxr) [reply pyupgrade.py]: I wanted to keep the order, will use `list(OrderedDict.from_keys(...))`

File: tests/oserror_aliases_test.py (hunk 2 above)
Comment (mxr) [reply tests/oserror_aliases_test.py]: sure

**Comment by mxr** [comment]
pushed up 🙏

**Comment by asottile** [comment]
can you add this test in the noop category:

```python
try:
    pass
except (
    T,
):
    pass
```

Also maybe split the formatted tests out from the non-formatted tests -- every non-formatted test added to the noop test adds 3 identical tests

**Comment by mxr** [comment]
Sure. I added the tests, added some test cases with multiple excepts, and reorganized the code a bit.

**Pull request merged**